packages/travix-ui-kit/components/messageBox/messageBox.js

```diff
@@ -29,6 +29,10 @@ const MessageBox = (props) => {
     </div>
   );
 
+  if (!(header || children)) {
+    return null;
+  }
```

Comment:
Additionally to this, would it be wise to create a conditional custom propType for both `title` and `children` props? I mean something like:

```js
title: (props, propName, componentName) => {
  if (!props.children && !props[propName]) {
    return new Error('Please provide a children or title prop')
  }
}
```

Reply:
No, it's ok when we don't have label in some cases, for some affiliates.